api/proto/disperser/disperser.proto

```diff
@@ -11,6 +11,12 @@ service Disperser {
 	// processing status of the blob.
 	rpc DisperseBlob(DisperseBlobRequest) returns (DisperseBlobReply) {}
 
+	// This API require valid payments to accept blob to disperse from clients.
+	// This executes the dispersal async, i.e. it returns once the request
+	// is accepted. The client could use GetBlobStatus() API to poll the the
+	// processing status of the blob.
+	rpc DispersePaidBlob(DispersePaidBlobRequest) returns (DisperseBlobReply) {}
```

Comment:
should we implement a canned method in disperser server which returns `Unimplemented` error? ex. https://github.com/Layr-Labs/eigenda/blob/master/disperser/apiserver/server_v2.go#L39-L41

Reply:
yep, I will add

Reply:
Second thought is we don't need to add this in server v2, because `DisperseBlobRequest` contains `common.v2.BlobHeader` which contains `common.PaymentHeader`. So I think `DisperseBlobRequest` API should be able to handle payments and no additional API is needed

Reply:
yeah i meant to add it in v1 server

Reply:
added to v1